### DISCUSSION TITLE
How to disable the phone-home features?

### DISCUSSION THREAD

**Discussion by benallard** [jobrunr/jobrunr#584] [Q&A]
I just figured out jobrunr hints me every few hours that a new version is available. Seems like a simple GET to `github.com`. And then I read from the release note that regularly the amount of successful jobs is posted to some central server.

Is there more I should be aware of ?

Is there a way to disable both (and any other ones that might come up in the future?)?

Thanks.

**Comment by rdehuyss** [comment]
Before, I answer (which can be found in the docs), can I ask you why it is disturbing you? 

JobRunr only shares some anonymous information like how many jobs where succeeded, the amount of background job servers.

**Comment by benallard** [comment]
There seems to be a way to disable the sending of sucessful job amount using quarkus / spring-boot or micronaut, but not using the fluent API, beside I didn't found a way to disable the check for a newer version.

I'd like to have one settings to completely prevent outgoing connections to external servers.

The application we are running is in a very strict environment, and the operator doesn't like information about its application leaking outside.

**Reply by rdehuyss** [reply]
Well, the reason for the check for a new version is that otherwise people log too many bugs from older versions which was too time consuming for me.

I may allow the opt-out for paying customers in the pro version but the version checks definitely stays in the free version. Also, from what I know, most people like to know that a new version is available.

The opt-out for the statistics is also documented in the Javadoc for the JobRunr Configuration.
